### PR TITLE
[OYPD-310] Camp menu links, default home link to camp.

### PR DESCRIPTION
**Pull request by Kerby70**
Jira https://propeople-us.atlassian.net/browse/OYPD-310
D.o https://www.drupal.org/node/2857094

Steps to review
- [x] Create landing page
- [x] Save and publish
- [x] Create camp with a camp menu
  - [x] Include a link to the landing page as a menu link
- [x] Save and publish
- [x] See home link, that links to the camp, and has the 'active' class.
- [x] Edit the landing page
  - [x] Add a camp menu
- [x] Save and publish
- [x] See the camp menu with home link that does not have any class, and landing page link that has the 'active' class

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
<strong>4</strong>  errors for CodeSniffer: Drupal standard file http://openy-dev.ffwua.com/build634/build_reports/Drupalsniff.txt
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build634 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build634 |
| Installation process | http://install.openy-dev.ffwua.com/build634/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/634/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build634/profiles/openy/build/reports/behat

File: modules/openy_features/openy_prgf/modules/openy_prgf_camp_menu/src/Plugin/Block/CampMenu.php [hunk 1]
```diff
@@ -96,9 +96,18 @@ public function build() {
     $current_route_url = Url::fromRoute($current_route->getRouteName(), $current_route->getRawParameters()->all());
     $current_internal_path = $current_route_url->getInternalPath();
 
+    // Add default home link to the camp node.
+    array_unshift($links, [
+        'uri' => 'entity:' . $camp->getEntityTypeId() . '/' . $camp->id(),
+        'title' => 'Home',
```
Comment (alexschedrov) [inline modules/openy_features/openy_prgf/modules/openy_prgf_camp_menu/src/Plugin/Block/CampMenu.php]: Should be translatable.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
<strong>4</strong>  errors for CodeSniffer: Drupal standard file http://openy-dev.ffwua.com/build638/build_reports/Drupalsniff.txt
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build638 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build638 |
| Installation process | http://install.openy-dev.ffwua.com/build638/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/638/

**Comment by alexschedrov** [comment]
1. Please fix code sniffer.

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build638/profiles/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build647 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build647 |
| Installation process | http://install.openy-dev.ffwua.com/build647/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/647/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build647/profiles/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build649 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build649 |
| Installation process | http://install.openy-dev.ffwua.com/build649/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/649/

**Review by alexschedrov — APPROVED** [review]
Approved.

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build649/profiles/openy/build/reports/behat

**Pull request merged**